docs/dev/README.md

```diff
@@ -1,35 +1,73 @@
 
-Metrics reported from accepted integrations are not counted as custom metrics, and therefore don't impact your custom metric allocation. (Integrations that emit potentially unlimited metrics may still be considered custom.) Ensuring native support for Datadog reduces friction to adoption, and incentivizes people to use your product, service, or project. Also, being featured within the Datadog ecosystem is a great avenue for added visibility.
+## Request a sandbox account
 
-### What's the process?
+All Technology Partners can request a dedicated sandbox Datadog account to aid in their development. This sandbox account has a free license that you can use to send in data, build out dashboards, and more. 
 
-The initial goal is to generate some code that collects the desired metrics in a reliable way, and to ensure that the general integration framework is in place. Start by writing the basic functionality as a custom Check, then fill in the framework details from [Create a New Integration][2].
+To request a sandbox account:
 
-Next, open a pull request against the [integrations-extras repository][3]. This signals to Datadog that you're ready to start reviewing code together. Don't worry if you have questions about tests, Datadog internals, or other topics - the integrations team is ready to help, and the pull request is a good place to go over those concerns. Be sure to take advantage of the [Community Office Hours][4] as well!
+1. Log into the [Datadog Partner Portal][6].
+2. On your personal homepage, click on the **Learn More** button under **Sandbox Access**.
+3. Select **Request Sandbox Upgrade**.
 
-Once the integration has been validated (functionality, framework compliance, and general code quality) it is merged into Extras. Once there, it becomes part of the Datadog ecosystem. Congratulations!
+<div class="alert alert-info">If you are already a member of a Datadog organization (including a trial org), you may need to switch to your newly created sandbox. For more information, see the <a href="https://docs.datadoghq.com/account_management/org_switching/">Account Management documentation</a>.</div>
 
-### What are your responsibilities?
+Creating a developer sandbox may take up to one or two business days. Once your sandbox is created, you can [invite new members from your organization][7] to collaborate with.
 
-Going forward, you, as the author of the code, are the active maintainer of the integration. You're responsible for maintaining the code and ensuring the integration's functionality. There is no specific time commitment, but you must be a maintainer and take care of the code for the foreseeable future. Datadog extends support on a best-effort basis for Extras, so you are not alone.
+## Let's Get Started 
+To create an Agent-based integration, [click here][2].
 
-## Let's get started!
+To create an API integration, [click here][1].
```

Comment:
Might be worth calling out the Marketplace here if we intend for this to be the landing page for partners because we also offer software license listings and professional services.